glean-core/src/metrics/mod.rs

```diff
@@ -2,6 +2,10 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+//! # Metrics
+//!
+//! Glean supports different metric types to store data.
```

Comment:
nit: should this be more like "The different metric types supported by the Glean SDK to handle data"?